git_machete/github.py

```diff
@@ -22,7 +22,7 @@
 GITHUB_DOMAIN = "github.com"
 GITHUB_REMOTE_PATTERNS = [
     "^https://github\\.com/(.*)/(.*)\\.git$",
-    "^git@github\\.com:(.*)/(.*)\\.git$",
+    "^(?:ssh:\\/\\/)?git@github\\.com(?:\\/|:)(.*)/(.*)\\.git$"
```

Comment:
Ouch, sorry, I didn't notice the subtle difference other than `ssh://` prefix... also forgot about the non-capturing groups

Reply:
So indeed, if you could revert your original version, that was much more readable indeed!